guides/common/modules/proc_removing-packages-from-a-host.adoc

```diff
@@ -1,11 +1,13 @@
 [id="removing-packages-from-a-host_{context}"]
 = Removing Packages from a Host
 
-You can remove packages from a host in the {ProjectWebUI}.
+You can remove a single package or multiple packages from a host in the {ProjectWebUI}.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
 . Click the name of the host you want to modify.
 . Click the *Content* tab, then click the *Packages* tab.
 . Click the vertical ellipsis for the package you want to remove and select *Remove*.
-You get a REX job notification once the remote execution job is complete.
\ No newline at end of file
+** To remove multiple packages, select the packages by checking the boxes and clicking *Remove* from the vertical ellipsis at the top.
```

Comment:
If you want to make the workflow for multiple packages, it should be rewritten properly, i.e. it should look more like the original Upgrading packages procedure.

Reply:
Completely changed the workflow in both. :+1: